tests/e2e/slashing.go

```diff
@@ -441,12 +446,13 @@ func (suite *CCVTestSuite) TestHandleSlashPacketDistribution() {
 	// the test cases verify that only the unbonding tokens get slashed for the VSC ids
 	// mapping to the block heights before and during the undelegation otherwise not.
 	testCases := []struct {
+		name     string
 		expSlash bool
 		vscID    uint64
 	}{
-		{expSlash: true, vscID: vscIDs[0]},
-		{expSlash: true, vscID: vscIDs[1]},
```

Comment:
@smarshall-spitzbart I think this needs to be redone on my part. I think the point of the test got lost, right?

In this version, the second slash cannot be applied since the SlashAck for given validator already exists. The slash acks are consumed (deleted) in endblock.

Reply:
Hmm it's not completely clear what the original intention of the test was, considering the comment: 

> The test cases verify that only the unbonding tokens get slashed for the VSC ids mapping to the block heights before and during the undelegation otherwise not.

I think Marius may be more familiar with this functionality. @mpoke do you know what this test is validating?

Reply:
Setting this to draft since difftests are not covering new functionality. Current difftest do not account for the fact that `LastDowntimeValsetUpdateId` must increase monotonically and sometimes creates traces where the valsetUpdateId in the slash packet is less than `LastDowntimeValsetUpdateId`.

Updating difftests will take will take an unknown amount of time for me to update.